src/sentry/release_health/duplex.py

```diff
@@ -692,6 +692,11 @@ def _dispatch_call_inner(
         )
 
         try:
+            # We read from the metrics source even if there is no need to compare.
+            metrics_fn = getattr(self.metrics, fn_name)
+            with timer("releasehealth.metrics.duration", tags=tags, sample_rate=1.0):
+                metrics_val = metrics_fn(*args)
+
```

Comment:
hm no, `should_compare` is now true all of the time (because the respective featureflag in flagr is). we may want to keep that featureflag to control load and (time) overhead in endpoints.

Reply:
The `organizations:release-health-check-metrics` feature flag is always true, but `should_compare` depends mainly on the `metrics_start` timestamp, e.g. https://github.com/getsentry/sentry/blob/7ee396c3d0c2c0d15d74327cd034690a9e06c386/src/sentry/release_health/duplex.py#L777

Reply:
ah you're right. perhaps we should check the featureflag separately (split up `should_compare`)

but this is also fine